Suggest specifying font-size and margin-block on h1 (#37827)
mdn/content: files/en-us/web/html/element/heading_elements/index.md

Prefer using only one `<h1>` per page and [nest headings](#nesting) without skipping levels.

> ### Specify `font-size` and `margin-block` on `<h1>`
> 
> The HTML standard specifies that `<h1>` elements in a `<section>`, `<article>`, `<aside>`, or `<nav>` element should render as an `<h2>` (smaller {{cssxref("font-size")}} and {{cssxref("margin-block")}}), or as an `<h3>` if nested another level, and so on. There is a [proposal](https://github.com/whatwg/html/issues/7867) to remove this special default style, so that `<h1>` always has the same default style (as it was originally).

Reviewer: ```suggestion
### Specifying a uniform font size  for `<h1>`

The [HTML standard](https://html.spec.whatwg.org/multipage/rendering.html#sections-and-headings) specifies that `<h1>` elements in a `<section>`, `<article>`, `<aside>`, or `<nav>` element should render as an `<h2>` (smaller {{cssxref("font-size")}} with an adjusted {{cssxref("margin-block")}}), or as an `<h3>` if nested another level, and so on.
```
We don't know if the spec will change, so let's just link to the spec.
The margin increases in `em` as the size of the `em` decreases, so "adjusting" seems like what is happening (versus "smaller").
Suggest a title change to why we are suggesting the section rather than what we are doing code wise.

Author: I'd like to keep a link to the proposed change, since it's a deprecation of sorts. Maybe in a note or similar?

Author: How about this:

```suggestion
### Specifying a uniform font size  for `<h1>`

The [HTML standard](https://html.spec.whatwg.org/multipage/rendering.html#sections-and-headings) specifies that `<h1>` elements in a `<section>`, `<article>`, `<aside>`, or `<nav>` element should render as an `<h2>` (smaller {{cssxref("font-size")}} with an adjusted {{cssxref("margin-block")}}), or as an `<h3>` if nested another level, and so on.

> [!NOTE]
> There is a [proposal](https://github.com/whatwg/html/issues/7867) to remove this special default style, so that `<h1>` always has the same default style. This proposal is currently [implemented in Firefox Nightly](/en-US/docs/Mozilla/Firefox/Experimental_features#ua_styles_for_h1_nested_in_sectioning_elements).
```

> To avoid getting smaller `<h1>`, and to avoid the rendering changing when browsers remove the special `<h1>` default styling, use the following style rule:

Reviewer 2: ```suggestion
To ensure consistent `<h1>` rendering, use the following style rule:
```
